users/vieting/experiments/switchboard/transducer/feat/experiments.py

```diff
@@ -277,6 +287,114 @@ def run_rasr_gt_stage1():
     return nn_system, report
 
 
+def run_scf_stage1():
+    ctc_alignment = get_ctc_alignment(ctc_alignment_model="scf", alignment_epoch=400)
+
+    gs.ALIAS_AND_OUTPUT_SUBDIR = "experiments/switchboard/transducer/feat/"
+    _, dev_corpora, _ = get_switchboard_data()
+    returnn_datasets = get_returnn_datasets_transducer_viterbi(context_window={"classes": 1, "data": 121})
+    returnn_datasets_hash_break = get_returnn_datasets_transducer_viterbi(
+        context_window={"classes": 1, "data": 121},
+        keep_hashes=False,
+    )
+    returnn_datasets_align_ctc = get_returnn_datasets_transducer_viterbi(
+        alignment=ctc_alignment,
+        features="waveform_pcm",
+    )
+    returnn_args = {
+        "batch_size": 15000,
+        "datasets": returnn_datasets,
+        "extra_args": {
+            # data sequence is longer by factor 4 because of subsampling and 80 because of feature extraction vs.
+            # raw waveform
+            "chunking": ({"classes": 64, "data": 64 * 4 * 80}, {"classes": 32, "data": 32 * 4 * 80}),
+            "gradient_clip": 20.0,
+            "learning_rate_control_error_measure": "sum_dev_score",
+            "min_learning_rate": 1e-6,
+            "learning_rate_n_step": 2650,
+            "extern_data": {
+                "data": {"dim": 1, "dtype": "int16"},
+                "classes": {"dim": 88, "dtype": "int8", "sparse": True},
+            },
+            "min_chunk_size": {"classes": 2, "data": 644},
+        },
+        "specaug_old": {"max_feature": 15},
+    }
+
+    feature_args = {
+        "class": "ScfNetwork",
+        "size_tf": 256 // 2,
+        "stride_tf": 10 // 2,
+        "preemphasis": 0.97,
+        "wave_norm": True,
+        "wave_cast": True,
+    }
+    common_args = {
+        "feature_args": feature_args,
+        "lr_args": {"dynamic_learning_rate": dynamic_learning_rate_configurable},
+    }
+
+    _, nn_system_ctc = run_scf_baseline_ctc()
+
+    existing_model_dict = {
```

Comment:
I would also include the `"existing-model"` level in here. That makes the definition of the experiments below easier to read.
```suggestion
    preload_dict = {
```